proto/zetachain/zetacore/crosschain/outbound_tracker.proto

```diff
@@ -12,5 +12,5 @@ message OutboundTracker {
   string index = 1; // format: "chain-nonce"
   int64 chain_id = 2;
   uint64 nonce = 3;
-  repeated TxHashList hash_list = 4;
+  repeated TxHash hash_list = 4;
```

Comment:
This change might need a migration script to manage migrating any existing outbound trackers to the new format when the upgrade happens

Reply:
Thanks for the comment!
Can you please share if there is any reference that I can use? Or maybe explain a bit more so that I can implement that on my own.

Reply:
You don't need migration script here if you don't change data types